scripts/processing/08-make_cov.py

```diff
@@ -30,7 +31,8 @@ def run_covariance(subject_id, tsss=False):
                             % (subject, l_freq))
     print('  Computing regularized covariance')
     epochs = mne.read_epochs(fname_epo, preload=True)
-    cov = mne.compute_covariance(epochs, tmax=0, method='shrunk')
+    cv = KFold(3, random_state=random_state)
```

Comment:
is it really necessary. For me it's deterministic by default. can you confirm?

Reply:
yes I think it's deterministic by default but I added it just to be explicit

Reply:
I'd rather add a comment saying this uses CV but it's deterministic, rather than add code (a comment would be needed even with the code to explain why, otherwise people might assume it is necessary to get KFold CV)

Reply:
could it explain the differneces on results between yours and mainak?

Reply:
+1

Reply:
umm ... I actually prefer to leave it like this because we don't have tests. I can add a comment to explain why it's like this though. When I see `cv=3` it doesn't tell me if it's deterministic or not. I have to either look at the code or go to scikit-learn documentation ...